### DISCUSSION TITLE
Automazioni con alcuni sensori

### DISCUSSION THREAD

**Discussion by gmarongiu71** [indomus/forum#1667] [GENERALE - Discussioni generiche]
Ho un problema con il sensore della temperatura che è derivato da un climate perchè ho dovuto inserire un fattore correttivo.
Se utilizzo lo strumento sviluppatori i valori sono corretti, ma non mi attiva l'automazione di seguito. 
Per testing ho inserito i valori più alti dei reali dicendo che se sono minori far attivare l'automazione. Le notifiche funzionano perchè le uso già altrove.
Grazie a chi mi aiuta.

Nella foto il valore della lettura tramite strumento sviluppatori
![Hassio](https://user-images.githubusercontent.com/103220010/165905168-7b6ab6f6-4ae1-4f47-975f-6d79db27aeb9.jpg)

Temperatura Acquario (Sotto i 50° è voluto per testare il trigger):
![acquario](https://user-images.githubusercontent.com/103220010/165906746-9296a3c5-df06-4b15-9de5-53a1b2942f29.jpg)

**Comment by 50494554524F** [comment]
indizio: trigger, attribute...

**Reply by gmarongiu71** [reply]
sotto trigger non trovo attribute :(
Grazie

**Reply by 50494554524F** [reply]
hai messo   "attribute: unit_of_measurement", con questo non funziona...

**Reply by gmarongiu71** [reply]
E' configurato in questo modo, ho messo minore di 50 per provare l'automazione, poi inserirò maggiore di 26... altrimenti li faccio bolliti :)


![Temp](https://user-images.githubusercontent.com/103220010/166305981-949bc030-74c1-4563-963a-200b1aea40ce.png)

**Reply by 50494554524F** [reply]
rileggi qui

https://www.home-assistant.io/docs/automation/trigger/#triggering-on-attribute-changes

come l'hai messa controlla se cambia l'attribute unit_etc, ma quello non cambierà mai ed in ogni caso non è quello che ti interessa controllare...

**Reply by gmarongiu71** [reply]
Sono neofita.... cosa dovrei inserire per leggere quel dato? Grazie

**Reply by gmarongiu71** [reply]
Mi fa scegliere tra tre opzioni che sono, oltre quella, icon e friendly name

**Reply by 50494554524F** [reply]
lascia attributo vuoto...

**Reply by gmarongiu71** [reply]
non risolve.... premetto che il sensore è derivato perchè mi segnava una temperatura per 10 e quindi ho dovuto correggerlo. In pratica al poso di 25 gradi leggeva 250

**Reply by 50494554524F** [reply]
porta la temperatura sopra 50 e poi riabbasala, solo così scatterà... per fare le prove puoi usare un input_number e mettere i valori a mano

**Reply by gmarongiu71** [reply]
Da modelli in Strumenti per sviluppatori?

**Comment by KC-inDomus** [comment]
https://indomus.it/formazione/esercizi-di-domotica-su-home-assistant-2-definire-diversi-trigger-per-innescare-automazioni/

**Reply by gmarongiu71** [reply]
Grazie

**Reply by gmarongiu71** [reply]
Il trigger, lasciando quel campo vuoto ha funzionato... Grazie :)
